CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # edge-react-gui
 
+## 2.17.0 (2022-06-22)
+
+- Replace edge-currency-bitcoin with more performant edge-currency-plugins based on bitcoin.js
+- Add Litecoin and Dash as default wallets
+- Add import seed for bitcoin-based currencies
+- Deprecate Changelly swap plugin
+- Deprecate SMART, EBST, and FTC
+- Improve list-scrolling performance
+- Refactor the Scan Modal
+- Refactor the WalletListSortModal to use the ListModal
+- Refactor the SceneHeader into a memoized functional component
+- Refactor CurrencySettingsScene to pull directly from plugin currencyInfos
+- Update ExchangeQuoteComponent and remove unused DataRow and CardContent
+- Use theme hooks instead of higher order components
+- Fix deep link listener instability
+- Fix incorrect fiat display until restart for new wallets
+- Fix Invalid Password text alignment
+- Switch BSV block explorer to WhatsOnChain.com
+- Decrease swipeable row sensitivity
+- Add edge-currency-plugins v1.0.0
+- Remove edge-currency-bitcoin
+- Upgrade edge-core-js to v0.19.20
```

Comment:
You also need to include the changed from 0.19.19 since the last version of core on master in 0.19.18

Reply:
It is included it's just one line

Reply:
My bad, I thought there were more changes. Never mind.